src/deepmol/models/deepchem_models.py

```diff
@@ -199,17 +202,42 @@ def predict_on_batch(self, dataset: Dataset) -> np.ndarray:
         """
         return super(DeepChemModel, self).predict(dataset)
 
-    def save(self):
+    def save(self, file_path: str = None):
         """
-        Saves deepchem model to disk using joblib.
+        Saves deepchem model to disk.
+
+        Parameters
+        ----------
+        file_path: str
+            Path to the file where the model will be stored.
         """
-        save_to_disk(self.model, self.get_model_filename(self.model_dir))
+        if file_path is None:
```

Comment:
Why make file_path optional and then raise a ValueError if is not set? Wouldnt be simple just to make it mandatory?

Reply:
yes, you're right